src/coreclr/src/pal/inc/palprivate.h

```diff
@@ -209,12 +175,6 @@ GetModuleFileNameA(
     OUT LPSTR lpFileName,
     IN DWORD nSize);
 
-
-PALIMPORT
```

Comment:
Delete FreeEnvironmentStringsA as well? It comes with GetEnvironmentStringsA

Reply:
There is palinternal.h that seems to be have number of other methods that are only used by PAL tests and nothing else.

Reply:
Deleted `FreeEnvironmentStringsA`.

Some of these which are not directly used outside of PAL and are not yet deleted have the implementation in such a way that their `W` counterpart method is the caller. For example `CreateDirectoryW` accepts `LPWSTR`, then narrows it to `LPSTR` and passes it to `CreateDirectoryA` for the actual operation (which calls libc/OS APIs that typically accept utf8 `char*`/`LPSTR`). Similar cases are SetEnvironmentVariableA, CopyFileA, MoveFileA etc. In addition, `CopyFileA` has one caller outside of PAL in `src/coreclr/src/ToolBox/superpmi/superpmi/jitinstance.cpp:`, which can be inlined like we have done for `CreateProcessA` here.

I think we can still delete their tests, their header declarations, their exports (in `mscordac_unixexports.src` etc.) and then either:

* leave their implementations in local file, or
* inline their implementation inside the `W` methods